security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -382,6 +382,7 @@
                         <dns_cx>CloudXNS.com API</dns_cx>
                         <dns_cn>Core-Networks API</dns_cn>
                         <dns_cyon>cyon.ch API</dns_cyon>
+                        <dns_dedyn>deSEC.io API</dns_dedyn>
```

Comment:
The acme.sh dns hook is "dns_desec", all fields should use this name instead of "dns_dedyn".

Reply:
@fraenki, thanks for review! It is fixed now.